lang/pillow/patches/010-fix-paths.patch

```diff
@@ -0,0 +1,180 @@
+diff -rupN Pillow-3.2.0.orig/setup.py Pillow-3.2.0/setup.py
+--- Pillow-3.2.0.orig/setup.py	2016-04-01 14:17:23.000000000 +0200
++++ Pillow-3.2.0/setup.py	2016-04-28 23:24:08.633969175 +0200
+@@ -90,11 +90,11 @@ except (ImportError, OSError):
+ NAME = 'Pillow'
+ PILLOW_VERSION = '3.2.0'
+ TCL_ROOT = None
+-JPEG_ROOT = None
++JPEG_ROOT = _lib_include(os.environ['STAGING_DIR'] + "/usr")
+ JPEG2K_ROOT = None
+-ZLIB_ROOT = None
+-TIFF_ROOT = None
+-FREETYPE_ROOT = None
++ZLIB_ROOT = _lib_include(os.environ['STAGING_DIR'] + "/usr")
++TIFF_ROOT = _lib_include(os.environ['STAGING_DIR'] + "/usr")
++FREETYPE_ROOT = _lib_include(os.environ['STAGING_DIR'] + "/usr")
+ LCMS_ROOT = None
+ 
+ 
+@@ -187,129 +187,6 @@ class pil_build_ext(build_ext):
+             _add_directory(library_dirs, os.path.join(prefix, "lib"))
+             _add_directory(include_dirs, os.path.join(prefix, "include"))
+ 
+-        #
+-        # add platform directories
+-
+-        if sys.platform == "cygwin":
```

Comment:
To simplify the patch it maybe makes sense to add a dummy `if` clause here, something like:

    if os.environ['STAGING_DIR']:
        print("Not altering search path")
    elif sys.platform == "cygwin":
        ...